### PR TITLE
Add a test covering that this infamous query does not crash Spark

### PR DESCRIPTION
**Pull request by exyi**
Turns out it doesnt crash it here, so the error must be in jetfire.

### CODE REVIEW

File: src/test/scala/cognite/spark/v1/RawTableRelationTest.scala [hunk 1]
```diff
@@ -502,4 +502,41 @@ class RawTableRelationTest
...
+            ''') as json
+            FROM table_which_does_not_exist
+          )
+        select first(externalId)
+        from (select json.externalId as externalId from raw_table)
+        GROUP BY concat_ws(':', externalId, externalId)
+        """
+
+    val exception = sparkIntercept(spark.sql(query).collect())
+    assert(exception.getMessage.contains("Following databases not found: datybasy."))
```
Comment (vuhainguyen) [inline src/test/scala/cognite/spark/v1/RawTableRelationTest.scala:536]:
```suggestion
    val exception = the[CdpApiException] thrownBy {
      spark.sql(query).collect()
    }
    assert(exception.getMessage.contains("Following databases not found: datybasy"))
```

**Review by vuhainguyen — APPROVED** [review]

File: src/test/scala/cognite/spark/v1/RawTableRelationTest.scala [hunk 2]
```diff
@@ -502,4 +502,41 @@ class RawTableRelationTest
...
+    val query =
+      """
+        WITH
+          raw_table AS (
+            SELECT
+            from_json(columns,
+            '''
+            externalId string
+            ''') as json
+            FROM table_which_does_not_exist
```
Comment (emelvarol-cognite) [inline src/test/scala/cognite/spark/v1/RawTableRelationTest.scala:528]: is this the same as `cdf_raw('db_which_does_not_exist', 'table_which_does_not_exist')`

Comment (emelvarol-cognite) [reply src/test/scala/cognite/spark/v1/RawTableRelationTest.scala:528]: aha sorry got it now

**Comment by emelvarol-cognite** [comment]
Like the branch name...

**Pull request merged**